spec/src/main/asciidoc/ServerPages.adoc

```diff
@@ -1906,13 +1856,12 @@ starting with the prefixe `jakarta` are reserved by the
 different specifications of the Jakarta EE platform. The `jakarta.servlet`
 prefix is reserved and used by the servlet and JSP specifications.
 
-==== [[a614]]Using JSPs as Error Pages
+==== Using JSPs as Error Pages
 
 A JSP is considered an Error Page if it sets
 the `page` directive’s `isErrorPage` attribute to `true` . If a page has
 `isErrorPage` set to `true` , then the “exception” implicit scripting
-language variable (see link:jsp.html#a748[See Implicit Objects
-Available in Error Pages.]) of that page is initialized. The variable is
+language variable (see <<Implicit-Objects-Available-in-Error-Pages>>) of that page is initialized. The variable is
```

Comment:
After I wrote this, I reviewing the rest of the document, and we seem to consistently use the `_` underscore for similarly styled links.

Reply:
I went back and forth on this , and looks like I missed one. I don't have a big preference one way or the other, I went with the `_` because asciidoc default id generation uses underscore so I figured that was at least consistent with default syntax: : [https://asciidoctor.org/docs/user-manual/#auto-generated-ids](https://asciidoctor.org/docs/user-manual/#auto-generated-idsl)

I'll replace the dashes here with _ unless there is an objection.